<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CChangeColors</class>
 <widget class="QDialog" name="CChangeColors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/nMSXtiles.ico</normaloff>:/res/nMSXtiles.ico</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="m_pGrColor0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color0.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>72</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color3.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>16</number>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>32</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color5.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>12</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color4.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>52</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color6.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>72</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color7.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>32</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color9.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>12</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color8.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>52</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color10.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>72</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color11.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor13">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>32</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color13.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange13">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>12</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color12.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange15">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor14">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>52</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color14.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="m_pGrColor15">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>72</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/new/res/res/color15.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange14">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pGrChange12">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="m_pGrOk">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>84</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pGrCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>84</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
